src/main/java/org/fenixedu/learning/domain/degree/DegreeSite.java

```diff
@@ -72,4 +94,19 @@ public void delete() {
         return Stream.of(categoryForSlug("announcement")).filter(Objects::nonNull).map(Category::makeWrap);
     }
 
+    private CMSFolder folderForPath(MenuContainer parent, String path) {
+
+        LocalizedString.Builder description = new LocalizedString.Builder();
```

Comment:
You can simply use `BundleUtil.getLocalizedString`, it has the same effect as this.